docs/hub/datasets-duckdb-cli-select.md

```diff
@@ -0,0 +1,150 @@
+# Query datasets
+
+Querying datasets is a fundamental step in data analysis. Here, we'll guide you through querying datasets using various methods.
+
+There are several [different ways](https://duckdb.org/docs/data/parquet/overview.html) to select your data.
+
+Using the `FROM` syntax:
+```bash
+FROM 'hf://datasets/jamescalam/world-cities-geo/train.jsonl' SELECT city, country, region LIMIT 3;
+
+┌────────────────┬─────────────┬───────────────┐
+│      city      │   country   │    region     │
+│    varchar     │   varchar   │    varchar    │
+├────────────────┼─────────────┼───────────────┤
+│ Kabul          │ Afghanistan │ Southern Asia │
+│ Kandahar       │ Afghanistan │ Southern Asia │
+│ Mazar-e Sharif │ Afghanistan │ Southern Asia │
+└────────────────┴─────────────┴───────────────┘
+
+```
+
+Using the `SELECT` and `FROM` syntax:
+
+```bash
+SELECT city, country, region FROM 'hf://datasets/jamescalam/world-cities-geo/train.jsonl' USING SAMPLE 3;
+
+┌──────────┬─────────┬────────────────┐
+│   city   │ country │     region     │
+│ varchar  │ varchar │    varchar     │
+├──────────┼─────────┼────────────────┤
+│ Wenzhou  │ China   │ Eastern Asia   │
+│ Valdez   │ Ecuador │ South America  │
+│ Aplahoue │ Benin   │ Western Africa │
+└──────────┴─────────┴────────────────┘
+
+```
+
+Count all parquet files matching a glob pattern:
+
+```bash
+SELECT COUNT(*) FROM 'hf://datasets/jamescalam/world-cities-geo/*.jsonl';
+
+┌──────────────┐
+│ count_star() │
+│    int64     │
+├──────────────┤
+│         9083 │
+└──────────────┘
+
+```
+
+You can also query Parquet files using the read_parquet and parquet_scan functions. Let's explore these functions using the auto-converted Parquet files from the same dataset.
+
+Select using [read_parquet](https://duckdb.org/docs/guides/file_formats/query_parquet.html) function:
+
+```bash
+SELECT * FROM read_parquet('hf://datasets/jamescalam/world-cities-geo@~parquet/default/**/*.parquet') LIMIT 3;
+```
+
+Read all files that match a glob pattern and include a filename column specifying which file each row came from:
+
+```bash
+SELECT * FROM read_parquet('hf://datasets/jamescalam/world-cities-geo@~parquet/default/**/*.parquet', filename = true) LIMIT 3;
+```
+
+Using [`parquet_scan`](https://duckdb.org/docs/data/parquet/overview) function:
```

Comment:
I think it's confusing, because it's only an alias to `read_parquet`. I would remove it.